CONTRIBUTING.md

```diff
@@ -89,57 +89,65 @@ Every bug fix should be accompanied with a unit test, so that we can prevent reg
 
 They are mostly welcome!
 
-## Operator SDK
+## Operator Lifecycle Manager (OLM)
 
-Coming soon, but for the moment:
+For production environments, it is recommended to use the [Operator Lifecycle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) to provision and update the OpenTelemetry Operator. Our operator is available in the [Operator Hub](https://operatorhub.io/operator/opentelemetry-operator), and when making changes involving those manifests the following steps can be used for testing. Refer to the [OLM documentation](https://sdk.operatorframework.io/docs/olm-integration/quickstart-bundle/) for more complete information.
 
-Build the operator
-```
-BUNDLE_VERSION=0.16.0
-make set-image-controller IMG=quay.io/jpkroehling/opentelemetry-operator
-make bundle
-make bundle-build BUNDLE_IMG=quay.io/jpkroehling/opentelemetry-operator-bundle:${BUNDLE_VERSION}
-podman push quay.io/jpkroehling/opentelemetry-operator-bundle:${BUNDLE_VERSION}
-opm index add --bundles quay.io/jpkroehling/opentelemetry-operator-bundle:${BUNDLE_VERSION} --tag quay.io/jpkroehling/opentelemetry-operator-index:${BUNDLE_VERSION}
-podman push quay.io/jpkroehling/opentelemetry-operator-index:${BUNDLE_VERSION}
-```
+### Setup OLM
+
+When using Kubernetes, install OLM following the [official instructions](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md). At the moment of this writing, it involves the following:
 
-Setup OLM (not needed on OpenShift):
 ```
-cd ${OLM_HOME}
-kubectl create -f deploy/upstream/quickstart/crds.yaml
-kubectl create -f deploy/upstream/quickstart/olm.yaml
+kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
 kubectl wait --for=condition=available deployment packageserver -n olm
 kubectl wait --for=condition=available deployment olm-operator -n olm
 kubectl wait --for=condition=available deployment catalog-operator -n olm
 ```
 
-Install the operator
+When using OpenShift, OLM is already installed.
+
+### Create the bundle and related images
+
+The following commands will generate a bundle under `bundle/` and build an image with its contents. It will then generate and publish an index image with the [Operator Package Manager (OPM)](https://github.com/operator-framework/operator-registry/blob/master/docs/design/opm-tooling.md#opm)
+
+```
+export VERSION=0.17.1
```

Comment:
nit: to avoid constant updates, should this just be `export VERSION=<required/latest version>

Reply:
I set it to `x.y.z`